src/shared/modules/discovery/discoveryDuck.ts

```diff
@@ -171,23 +196,76 @@ export const discoveryOnStartupEpic = (some$: any, store: any) => {
           discovered: onlyTruthy
         })
       }
+      const discoveryEndpoint = getDiscoveryEndpoint(
+        getHostedUrl(store.getState())
+      )
       return Rx.Observable.fromPromise(
         remote
-          .getJSON(getDiscoveryEndpoint(getHostedUrl(store.getState())))
+          .getJSON(action.discoveryURL || discoveryEndpoint)
           // Uncomment below and comment out above when doing manual tests in dev mode to
           // fake discovery response
           //Promise.resolve({
           // bolt: 'bolt://localhost:7687',
-          //neo4j_version: '4.0.3'
+          // neo4j_version: '4.0.3'
           //})
-          .then(result => {
+          .then(async result => {
+            const ssoProviders =
+              result.sso_providers || result.ssoproviders || result.ssoProviders
```

Comment:
again, might be nice to make this total case insensitive...

Reply:
I'll keep it like this for now as that's how it's done in bloom